requirements.txt

```diff
@@ -19,10 +19,12 @@ idna==2.10
 itsdangerous==1.1.0
 Jinja2==2.11.3
 jsonpatch==1.32
+ldap3==2.9
 Mako==1.1.2
 MarkupSafe==1.1.1
 marshmallow==3.6.0
 psycopg2==2.8.5
+pyasn1==0.4.8
```

Comment:
what's this one? I did not see it in ldap3 dependencies.

Reply:
It got installed with ldap3 and its the only dependency defined in their github: https://github.com/cannatag/ldap3/network/dependencies